readme.txt

```diff
@@ -3,22 +3,24 @@ Contributors: shawndh, markjaquith, mordauk, JustinSainton, wpsmith, bhwebworks,
 Tags: wp101, tutorials, video, help, learn, screencast
 Requires at least: 3.2
 Requires PHP: 5.4
-Tested up to: 4.9.5
-Stable tag: 4.1
+Tested up to: 5.1

```

Comment:
Was this version of the plugin tested up to 5.1?

Reply:
I've been testing it locally against trunk and [our Travis tests confirm it](https://travis-ci.org/liquidweb/wp101plugin/jobs/493945072).